Modul OOP/engine.py

```diff
@@ -0,0 +1,39 @@
+import models
+import settings
+import exceptions
+
+
+def get_player_name() -> str:
+    name = ''
+    while name == '':
+        name = (input("ENTER YOUR NAME: ")).strip()
```

Comment:
Зайві дужки ігноруються інтерпретатором.
Не помилка, але дуже ріже око.
```suggestion
        name = input("ENTER YOUR NAME: ").strip()
```